match2/commons/match_group_display_matchlist.lua

```diff
@@ -134,66 +134,57 @@ function MatchlistDisplay.Match(props)
 	DisplayUtil.assertPropTypes(props, MatchlistDisplay.propTypes.Match)
 	local match = props.match
 
+	local opponentWidth = math.floor(0.4 * props.width) - 1
+	local scoreWidth = 0.5 * (props.width - 5 - 2 * opponentWidth)
```

Comment:
[Magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants)?

Reply:
I added a comment explaining this